lab-justin/lib/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
```

Comment:
It was not covered in class, but I usually handle all of  process.arv in the main file of my program, by storing it in a variable. If you use process dot argv in other places in your code, it can cause side effects in the way people use and run your code....

I would have just passed in a variable to this function that was eather nothing or procsses.argv